editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1759,6 +1629,10 @@ AnimationPlayerEditor::AnimationPlayerEditor(AnimationPlayerEditorPlugin *p_plug
 
 	_update_player();
 
+	library_editor = memnew(AnimationLibraryEditor);
+	add_child(library_editor);
+	library_editor->connect("update_editor", Callable(this, "_animation_player_changed"));
```

Comment:
This could use `callable_mp` no?

Reply:
I guess, but since this function had to be properly bound for the undo-redo, I just connected the classical way. No strong feeling either way.